nixos/tests/wordpress.nix

```diff
@@ -15,15 +15,12 @@ import ./make-test-python.nix ({ pkgs, ... }:
       services.httpd.adminAddr = "webmaster@site.local";
       services.httpd.logPerVirtualHost = true;
 
-      services.wordpress = {
-        # Test support for old interface
+      services.wordpress.sites = {
         "site1.local" = {
-          database.tablePrefix = "site1_";
+          database.tablePrefix = "site2_";
```

Comment:
any reason this is changed?

Reply:
I think we should keep both sites in the test, only adjust to use the final interface

Reply:
Good catch @mohe2015, fixed it!